aten/src/ATen/native/native_functions.yaml

```diff
@@ -2920,26 +2920,18 @@
   dispatch:
     CompositeImplicitAutograd: linear
     NestedTensorCPU, NestedTensorCUDA: nested_linear
+    MPS: _mps_linear
 
 - func: linear_backward(Tensor self, Tensor grad_output, Tensor weight, bool[3] output_mask) -> (Tensor, Tensor, Tensor)
   dispatch:
     NestedTensorCPU, NestedTensorCUDA: nested_linear_backward
-  autogen: linear_backward.out
```

Comment:
This was a rebase thing, autogen is a new keyword to me but I think it has to do with functionalization? maybe I need to add this back?

Will read up

Reply:
Ho yes, I didn't follow these PRs in detail. @bdhirsh would know :p

Reply:
#81437 seems to have introduced this, maybe adding some notes to the README.md file would be useful. From reading that PR though I think linear_backward satisfies the reqs for autogen. Will undo this removal.

Reply:
`autogen` definitely deserves a section in the README, sorry about that :(

The idea is that all (non-CompositeImplicitAutograd, non-view) ops in native_functions.yaml should now have both a functional and an out= variant. Instead of needing to write them all out, the `autogen` keyword is just a way to make it clear which of those missing NativeFunctions will get automatically created by the codegen.

The codegen actually figures out which ops are missing that it needs to generate entirely off of the schemas, and the `autogen` entry in native_functions.yaml is really just there to be explicit about which ops exist (so the codegen can also check that list against what it plans to create and error accordingly).